bin/devdata.py

```diff
@@ -0,0 +1,25 @@
+"""Copy tox environment variables from devdata."""
+
+import os
+from shutil import copy2, rmtree
+from subprocess import check_call
+from tempfile import mkdtemp
+
+
+def _get_devdata():
+    temp_dir = mkdtemp()
+    target = os.path.abspath(".devdata.env")
+
+    try:
+        check_call(["git", "clone", "git@github.com:hypothesis/devdata.git", temp_dir])
+        copy2(os.path.join(temp_dir, "via/devdata.env"), target)
+
+        print(f"Created {target}")
+
+    finally:
+        if os.path.isdir(temp_dir):
+            rmtree(temp_dir)
```

Comment:
You might want to copy-paste [how this is done in lms](https://github.com/hypothesis/lms/blob/e54cea31bdca810a78d772ebc357b449ed1022c3/lms/scripts/devdata.py#L107-L119) (it's also [done exactly the same way in h](https://github.com/hypothesis/h/blob/979a784005972d81e4d3eebcf75a6c7b746760fc/h/cli/commands/devdata.py#L145-L160)). The way it's done here seems to work fine, and I can't remember why I did it that exact way in lms and h, but the lms and h way is tested and proven, and there seems little point in having two different implementations of a hack like this to support

Reply:
Yep this is done now.